src/job-exporter/test/data/docker_inspect_amd.json

```diff
@@ -0,0 +1,337 @@
+                "/var/lib/kubelet/pods/a1310f10-5ea6-11ea-8a2a-90b11c27f535/volumes/kubernetes.io~secret/job-ssh-secret-volume:/usr/local/pai/ssh-secret:ro",
+                "/var/lib/kubelet/pods/a1310f10-5ea6-11ea-8a2a-90b11c27f535/etc-hosts:/etc/hosts",
+                "/var/lib/kubelet/pods/a1310f10-5ea6-11ea-8a2a-90b11c27f535/containers/app/8367221e:/tmp/pai-termination-log"
+            ],
+            "ContainerIDFile": "",
+            "LogConfig": {
+                "Type": "json-file",
+                "Config": {
+                    "max-file": "2",
+                    "max-size": "2g"
+                }
+            },
+            "NetworkMode": "container:087b43ac0b1c1465b11553550f249839e8d2d65936ca5678c55327bbee34e7e3",
+            "PortBindings": null,
+            "RestartPolicy": {
+                "Name": "no",
+                "MaximumRetryCount": 0
+            },
+            "AutoRemove": false,
+            "VolumeDriver": "",
+            "VolumesFrom": null,
+            "CapAdd": [
+                "SYS_ADMIN",
+                "IPC_LOCK",
+                "DAC_READ_SEARCH"
+            ],
+            "CapDrop": [
+                "MKNOD"
+            ],
+            "Capabilities": null,
+            "Dns": null,
+            "DnsOptions": null,
+            "DnsSearch": null,
+            "ExtraHosts": null,
+            "GroupAdd": null,
+            "IpcMode": "container:087b43ac0b1c1465b11553550f249839e8d2d65936ca5678c55327bbee34e7e3",
+            "Cgroup": "",
+            "Links": null,
+            "OomScoreAdj": -998,
+            "PidMode": "",
+            "Privileged": false,
+            "PublishAllPorts": false,
+            "ReadonlyRootfs": false,
+            "SecurityOpt": [
+                "apparmor=unconfined",
+                "seccomp=unconfined"
+            ],
+            "UTSMode": "host",
+            "UsernsMode": "",
+            "ShmSize": 67108864,
+            "Runtime": "rocm",
+            "ConsoleSize": [
+                0,
+                0
+            ],
+            "Isolation": "",
+            "CpuShares": 8192,
+            "Memory": 6989807616,
+            "NanoCpus": 0,
+            "CgroupParent": "/kubepods/poda1310f10-5ea6-11ea-8a2a-90b11c27f535",
+            "BlkioWeight": 0,
+            "BlkioWeightDevice": null,
+            "BlkioDeviceReadBps": null,
+            "BlkioDeviceWriteBps": null,
+            "BlkioDeviceReadIOps": null,
+            "BlkioDeviceWriteIOps": null,
+            "CpuPeriod": 100000,
+            "CpuQuota": 800000,
+            "CpuRealtimePeriod": 0,
+            "CpuRealtimeRuntime": 0,
+            "CpusetCpus": "",
+            "CpusetMems": "",
+            "Devices": [
+                {
+                    "PathOnHost": "/dev/fuse",
+                    "PathInContainer": "/dev/fuse",
+                    "CgroupPermissions": "rwm"
+                }
+            ],
```

Comment:
if this a real amd docker?  no gpu devices here

Reply:
An exception will raise during run `rocm-smi`. The related metrics will not export to `Prometheus`